README.rst

```diff
@@ -104,4 +116,4 @@ Contributing
 We welcome contributions. Please read the guide in `CONTRIBUTING.rst`_.
 
 .. _CONTRIBUTING.rst:
-   https://github.com/IntelPNI/brainiak/blob/master/CONTRIBUTING.rst
+   https://github.com/IntelPNI/brainiak/blob/master/CONTRIBUTING.rst
```

Comment:
No new line at end of file. Simply open the file in `vim` and save it and `vim` will add the newline automatically.